swift/ql/lib/codeql/swift/elements/KeyPathComponent.qll

```diff
@@ -1,6 +1,10 @@
 private import codeql.swift.generated.KeyPathComponent
 private import swift
 
+// the following QLdoc is generated: if you need to edit it, do it in the schema file
+/**
+ * A component of a `KeyPathExpr`.
+ */
```

Comment:
Not sure how I feel about this design.  I would really much rather edit it in the `.qll`, but then I can see the upside of using the same doc string as we already have and not getting out of sync.

Which classes get this treatment exactly?

Reply:
Getting the QLdoc from the `.qll` file would be technically possible, but challenging. And I can see it become quite a confusing mess, so I really prefer `schema.py` to be the source of truth here.

> Which classes get this treatment exactly?

The customized stubs (the hand-modified files defining the wrappers around `Generated::bla`, and only the classes derived from the generated one)